app/imdb_tools.py

```diff
@@ -83,27 +83,24 @@ def imdb_get_movies(config_path, plex, data):
         print_display = "| Processing: {}/{} {}".format(current_count, len(plex_movies), m.title)
         print(adjust_space(current_length, print_display), end="\r")
         current_length = len(print_display)
-        try:
```

Comment:
Does it break anything to leave `try/except`? I like the insurance.

Reply:
its just a good idea to only catch what we know is coming if an error like the occurs again we wouldn't have the error trace for the user without having to take it out again